src/plugins/condition/Condition.js

```diff
@@ -172,7 +186,7 @@ export default class ConditionClass extends EventEmitter {
         let found = this.findCriterion(criterion.id);
         if (found) {
             this.criteria[found.index] = criterion.data;
-            this.subscribe();
+            // this.subscribe();
```

Comment:
Remove this comment if it's not being used anymore.

Reply:
removed